use/adopter/setting-up-your-very-first-conversation.md

```diff
@@ -4,14 +4,14 @@
 
 ### 1.1. UCI Admin
 
-* Go to [Link](http://localhost:9097/uci-admin)
+* Go to admin console.
```

Comment:
Link admin console repo here.

Reply:
Also admin console has its own documentation. https://uci-web-channel-docs.vercel.app/

Reply:
> Also admin console has its own documentation. https://uci-web-channel-docs.vercel.app/

Hey, this is the web channel documentation and not the admin console documentation. I think the admin console documentation would be integrated in this repo itself.